oceannavigator/frontend/src/components/MainMap.jsx

```
@@ -0,0 +1,1740 @@
+
+  const show = (type, key) => {
+    resetMap();
+    props.updateState(["vectorId", "vectorType"], [key, type]);
+  };
+
+  const getBasemap = (source, projection, attribution) => {
+    switch (source) {
+      case "topo":
+        const shadedRelief = props.mapSettings.topoShadedRelief
+          ? "true"
+          : "false";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)